internal/controllers/operator_controller_test.go

```diff
@@ -665,51 +659,6 @@ var _ = Describe("Operator Controller Test", func() {
 				Expect(cond.Message).To(Equal("installation has not been attempted as resolution failed"))
 			})
 		})
-		When("the existing operator status is based on bundleDeployment", func() {
```

Comment:
As of https://github.com/operator-framework/operator-controller/pull/134, this block no longer has any specs in it. So it is unused. Therefore removing it.